app/javascript/delete_evaluator_modal.js

```diff
@@ -0,0 +1,77 @@
+export function initializeDeleteEvaluatorModal() {
+  let currentEvaluatorId, currentEvaluatorType, challengeId, phaseId;
+
+  document.querySelectorAll('[data-open-modal]').forEach(button => {
```

Comment:
do you want to refactor this to use [stimulus](https://stimulus.hotwired.dev/handbook/hello-stimulus) instead of vanilla js? 😇